README.md

```diff
@@ -152,6 +152,8 @@ Coming from `react-use`? Check out our
 
   - [**`useSyncedRef`**](https://react-hookz.github.io/web/?path=/docs/miscellaneous-usesyncedref--example)
     — Like `useRef`, but it returns immutable ref that contains actual value.
+  - [**`useCustomCompareMemo`**](https://react-hookz.github.io/web/?path=/docs/miscellaneous-useCustomCompareMemo--example)
+    — Like `useMemo` but with a callback to defines that factory function can be invoked.
```

Comment:
Like `useMemo` but uses provided comparator function to validate dependency changes.